### PR TITLE
feature: do not render decimal places on 'choose amount' screen of Sequoia template

### PR DESCRIPTION
**Pull request by ravinderk**
## Description

As per design Sequoia form template requirement, we want o to impose zero number for decimal on the frontend. Following PR resolve issue by returning zero on filter hook of helper function which developers use in there an add-on to access number of decimal admin setting.

## Affects

Change in PR will apply only to the frontend (because of template code loads in the frontend only) donation form only if it has a `Sequoia` form template or we are processing donation with this form or any ajax request on frontend related to this form.

## What to test

To test this PR, you can review a number of decimals apply to the amount in `Choose Amount` section on the donation form or amount on the receipt.
You can also test above mentioned item with `Fee Recovery` add-on.

## Screenshots:

image

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

File: src/Helpers/Form/Utils.php [hunk 1]
```diff
@@ -46,9 +46,9 @@ function isProcessingForm() {
  * @return bool
  */
 function isProcessingGiveActionOnAjax() {
-	return isset( $_REQUEST['action'] ) &&
-		   wp_doing_ajax() &&
-		   0 === strpos( $_REQUEST['action'], 'give_' );
+	$action            = isset( $_REQUEST['action'] ) ? give_clean( $_REQUEST['action'] ) : '';
+	$whiteListedAction = [ 'get_receipt' ];
+	return $action && wp_doing_ajax() && ( 0 === strpos( $action, 'give_' ) || in_array( $action, $whiteListedAction, true ) );
```
Comment (JasonTheAdams) [inline src/Helpers/Form/Utils.php:51]: I'm confused, is this related to limiting the decimal places?

Comment (ravinderk) [reply src/Helpers/Form/Utils.php:51]: @JasonTheAdams Yes, this is required. Basically form template was not loading when fetching receipt by ajax that's why hooks were not working. It will allow the loading form template on ajax. `get_reciept` is a special case in Give core where `action` parameter  (required for WP Ajax) does not have `give_` prefix.
